.github/workflows/deploy.yaml

```diff
@@ -30,6 +30,8 @@ jobs:
           cf-username: ${{ secrets.CF_USERNAME }}
           cf-password: ${{ secrets.CF_PASSWORD }}
           cf-org: ${{ secrets.CF_ORG }}
+          VENDOR_1: ${{ secrets.VENDOR_1 }}
+          VENDOR_2: ${{ secrets.VENDOR_2 }}
 
       - name: Deploy application
         run: cf push --vars-file vars.yaml
```

Comment:
Shouldn't the variable go in the deploy step?

Reply:
Good point, moved accordingly